cmd/skywire-cli/commands/rpc/root.go

```diff
@@ -0,0 +1,25 @@
+package clirpc
```

Comment:
Package may as well just be called `rpc` in my opinion and function could be called `Client()` This would make obsolete to use RPC as prefix for variables as well.